.github/workflows/check-health-daily.yml

```diff
@@ -21,12 +21,16 @@ jobs:
           target: wasm32-unknown-unknown
           override: true
           components: rustfmt, clippy
+    - name: Build
+      run: cargo build --verbose
+    - name: Run tests
+      run: cargo test --verbose
     - name: Create issue using REST API
       if: failure()
       run: |
         curl --request POST \
         --url https://api.github.com/repos/${{ github.repository }}/issues \
-        --header 'authorization: Bearer ${{ secrets.POLKADEX }}' \
+        --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
```

Comment:
Why did we make this change?

Reply:
right now automated issues are created by Gautham's account ... using GITHUB_TOKEN it will be created by a bot.